app/Models/EventPicture.php

```diff
@@ -43,11 +43,11 @@ class EventPicture extends Model
 
     protected $guarded = [];
 
-    public function user(){
+    public function user() {
```

Comment:
__[code_sniffer]__
* Opening brace should be on a new line

<!-- Comment by [SideCI](https://sideci.com) -->